Resources/Private/Language/locallang.xlf

```diff
@@ -97,6 +97,16 @@
 			<trans-unit id="widgets.documentation.label" xml:space="preserve">
 				<source>Check TYPO3 documentation</source>
  			</trans-unit>
+
+			<trans-unit id="dashboard.empty.title" xml:space="preserve">
+				<source>No dashboard configured :(</source>
```

Comment:
I guess this should be: No widgets on this dashboard :(

Reply:
updated